sponsors/notifications.py

```diff
@@ -0,0 +1,43 @@
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.conf import settings
+
+
+class AppliedSponsorshipNotification:
+    subject_template = None
+    message_template = None
+
+    def get_subject(self, context):
+        return render_to_string(self.subject_template, context).strip()
+
+    def get_message(self, context):
+        return render_to_string(self.message_template, context).strip()
+
+    def get_recipient_list(self, context):
+        raise NotImplementedError
+
+    def notify(self, user, sponsorship):
+        context = {'user': user, 'sponsorship': sponsorship}
+
+        send_mail(
+            subject=self.get_subject(context),
+            message=self.get_message(context),
+            recipient_list=self.get_recipient_list(context),
+            from_email=settings.DEFAULT_FROM_EMAIL,
+        )
```

Comment:
It's very likely that this class will become even more abstract so it can be re-used by the upcoming email notifications in the sponsorship process.